index.html

```diff
@@ -213,6 +208,36 @@ <h2 class="about-myself-title">About <br> Myself</h2>
                     </li>
                 </ul>          
         </section>
+
+      
+
+        <section class="contact-me">
+           
+            <h2>Contact me</h2>
+                <p class="form-text">
+                    If you have an application you are interested in developing,
+                    a feature that you need built or a project that needs coding.
+                    I’d love to help with it 
+                </p>
+            <form action="https://formspree.io/f/xayvqwwl" method="POST">                
+                <input type="text" id="name" name="user_name" placeholder="Jerry" required>                   
+                <input type="email" id="email"
+                 name="user_email" 
+                 placeholder="jerryowusu6133@gmail.com"
+                  required>                
+                <textarea
+                  id="msg"
+                  name="user_message"
+                  cols="30"
+                  rows="10"
+                  maxlength="500"
+                  placeholder="Write your message here"
+                  required>
```

Comment:
- [ ] In order to layout the elements on the page you should use Flexbox as required, please add `display: flex `on this section

Reply:
Done.